app/views/smart_proxies/tabs/_overview.html.erb

```diff
@@ -0,0 +1,7 @@
+<% tab.columns.each do |column| %>
+  <div class="col-md-<%= 12 / tab.columns.count %>">
+    <% column.widgets.each do |widget| %>
+      <%= render widget.partial, { :smart_proxy => smart_proxy, :widget => widget } %>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
What is the benefit of having extra attribute called column here? I mean, the extended template could contain the CSS class directly, therefore there was no need of maintaining columns. It makes sense for Dashboard Widgets where things are moveable/resizable.